### ISSUE TITLE
Picker component does not have an onTouch property

### ISSUE THREAD

**Issue by donatas-vindzigalskis** [react-native-community/react-native-picker#54]
# Bug

### Description:

The current picker does not have any event listeners for onBlur, onTouch events. So it is impossible in the app to know if the user has interacted with the picker component. The only way to know if a user has made contact with the component is through the onValueChange property. However, if the user opens the picker on Android, but doesn't select a value and/or closes the picker, the app cannot know if the user has interacted with the picker. 

This is needed for form validations, to display warnings or error messages after the user has "touched" the picker component.

## Environment info
System:
    OS: macOS 10.15.3
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
    Memory: 6.30 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 12.13.1 - /usr/local/bin/node
    npm: 6.13.4 - /usr/local/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.6010548
    Xcode: 11.3.1/11C504 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz => 0.61.4 
  npmGlobalPackages:
    create-react-native-app: 2.0.2
    react-native: 0.61.5

## Steps To Reproduce

1. Import Picker component from react-native library,
2. Check the props of Picker component.

## Expected Results

    There should be a listener event of touch gestures or focus changes.

**Comment by donatas-vindzigalskis** [comment]
This issue was moved from react-native library, as suggested by @TheSavior 's [comment](https://github.com/facebook/react-native/pull/28180#issuecomment-593568371), link to the [pull request raised](https://github.com/facebook/react-native/pull/28180) on react native repository.

**Comment by jainkuniya** [comment]
hi @donatas-vindzigalskis, can you try wrapping picker component with touchable components like `TouchableHighlight`?

**Comment by clamiax** [comment]
> hi @donatas-vindzigalskis, can you try wrapping picker component with touchable components like `TouchableHighlight`?

Just tried and doesn't works. I can't currently try with new `Pressable` API though.

**Comment by villanuevadani** [comment]
@clamiax did you manage to try that with Pressable?

**Comment by clamiax** [comment]
> @clamiax did you manage to try that with Pressable?

@villanuevadani No, I will need to upgrade my project which I can't do right now.

**Comment by tetianastepova** [comment]
I tried it with Pressable and no luck.

Does anyone know any workaround for this?
This is so strange that such an important thing is missing in library that is advised by react-native documentation...

**Comment by DaRizat** [comment]
+1 to this. Currently a real issue for me on form validations.

**Comment by programminPete** [comment]
@DaRizat  - that worked for me - thank you!  Note - I _did_ need to use the <View> wrapper; putting the responder events on the Picker itself didn't trigger

For my case, I have another button to advance screen after setting the value from <Picker />, but if someone pushed button very quickly after changing the picker it wouldn't set state correctly in time.  But using your wrapper technique I was able to disable the button while moving and re-enable once the onValueChange event fired - good stuff! thanks
